packages/rpc-methods/package.json

```diff
@@ -71,5 +71,11 @@
   "publishConfig": {
     "access": "public",
     "registry": "https://registry.npmjs.org/"
+  },
+  "lavamoat": {
+    "allowScripts": {
```

Comment:
Any idea why this was added now?

Reply:
no... not sure. I was prompted to run the `allow-scripts` script and did so and it created that diff. But I just removed it and reran the script and it was not added... Not sure what that's about.

Reply:
Removed the `allowScript` configuration here: https://github.com/MetaMask/snaps-monorepo/pull/1114/commits/90c5b479d2c8916fb391c693e3be621a51e2c27e